packages/project-disaster-trail/src/state/tasks.js

```diff
@@ -95,6 +164,17 @@ export const getTasksForEnvironment = createSelector(
   foundTasks => foundTasks
 );
 
+// helpful to derive the percentage complete of current time/total time
+// ie when a timer is started, current time is 0
+// as the timer 'ticks' the current time increases
+// and eventually reaches 100% complete
+// export const getPercentComplete = createSelector(
+//   ["tasks.activeTime", "tasks.startTime", "tasks.totalTime"],
+//   (activeTime, startTime, totalTime) => {
+//     return Math.max(0, Math.min(1, (activeTime - startTime) / totalTime)); // number within range of 0 - 1
+//   }
+// );
+
```

Comment:
Since we're using GSA, it looks like all this timer related stuff can be removed